### DISCUSSION TITLE
overlapping\duplicate grids from different offices

### DISCUSSION THREAD

**Discussion by dangelsaurus** [weather-gov/api#617] [Q&A]
I've come across a couple of almost perfectly overlapping grids from different forecast offices, with different (though similar) forecast.

https://api.weather.gov/points/44.9632,-118.2336 points to https://api.weather.gov/gridpoints/PDT/143,63/forecast
https://api.weather.gov/points/44.9644,-118.2319 points to https://api.weather.gov/gridpoints/BOI/76,158/forecast

however these two grids overlap almost perfectly. You see what amounts to maybe 10 ft of overlap

![image](https://user-images.githubusercontent.com/12521900/224460717-4c72e045-5e54-4b53-9cab-b45f87f264b3.png)

but the forecast are different, but I suspect that might be because they have different "updateTime'

Of the grids I monitor, I think I found maybe 11 overlapping\duplicates.  Since I obviously don't see all the grids, I was just curious if this was normal as a way to have some redundancy near borders of neighboring offices? a byproduct of how complicated breaking down areas into grids is? or just an error?

**Comment by jah1007** [comment]
Each forecast office’s forecast is made almost completely in a vacuum from the neighboring office on “normal” weather days. There are sanity checks that are done but there will be variations that occur because one forecaster believes one thing and the forecaster at the neighboring office believes something slightly different.
